views/editor/userbar.html

```diff
@@ -4,7 +4,7 @@
 
     <div id="navbar-left" class="container-flex">
       <div class="navbar-logo-container small-logo">
-        <a href="/{{ locale }}" class="logo" title="{{ gettext("backToHomepage") }}">
+        <a target="_blank" href="/{{ locale }}" class="logo" title="{{ gettext("backToHomepage") }}">
```

Comment:
I don't think we even need a target here... what would be the value of opening in a new tab/window for the logo? @flukeout?